BioForm/profiles/views.py

```diff
@@ -62,6 +61,37 @@ def logout_user(request):
     logout(request)
     return redirect("login")
 
+@login_required(login_url = 'login')
+def display_bio(request):
```

Comment:
Naming it `bio_view` will be better than `display_bio`, use the same convention for other views too.